internal/commands/run.go

```diff
@@ -105,6 +111,7 @@ func newRunCmd() *cobra.Command {
 		Args:  cobra.MaximumNArgs(1),
 	}
 
+	cmd.PersistentFlags().BoolVar(&dirty, "dirty", false, "mount project directory in read-write mode")
```

Comment:
There's no "true" or "false" on the UI side, it's a flag: you either set it or not.

```suggestion
	cmd.PersistentFlags().BoolVar(&dirty, "dirty", false, "if set the project directory will be mounted in read-write mode, otherwise the project directory files are copied, taking .gitignore into account")
```

What do you think about this instead?

Reply:
LGTM